flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/utils/AggregateUtil.scala

```diff
@@ -747,7 +747,7 @@ object AggregateUtil extends Enumeration {
   private[flink] def isTableAggregate(aggCalls: util.List[AggregateCall]): Boolean = {
     aggCalls
       .filter(e => e.getAggregation.isInstanceOf[AggSqlFunction])
-      .map(e => e.getAggregation.asInstanceOf[AggSqlFunction].makeFunction(null, null))
+      .map(e => e.getAggregation.asInstanceOf[AggSqlFunction].aggregateFunction)
```

Comment:
Hi @wuchong , the fix just have this line.
The hive function should been make by correct `constants` and `argTypes`. Otherwise it will throw an exception. (See `HiveAggSqlFunction`)
In this `isTableAggregate`, it just want to check the agg class type, so the better way is get the function instead of make a function.

Reply:
But from the exception stack attched in the JIRA, it seems that this is not where the exception throws.

Reply:
I think it is very old JIRA... At that time, it is a different root cause, but for now, it is a new root cause.
After add it case to hive, we can ensure the right logic.
I think I can update JIRA to new root cause. What do you think?

Reply:
I think it will be great to update the exception.

Reply:
updated.

Reply:
Thanks. I got the reason. Could you add a javadoc for `makeFunction` that it is only used for Hive UDF support, this is a temporary solusion and should be removed once FLIP-65 is finished.

Reply:
Updated.